packages/homepage/src/screens/home/experiment.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import { H2, P } from '../../components/Typography';
+
+import frameworks from '../../assets/images/frameworks.png';
+import things from '../../assets/images/things.png';
+import npm from '../../assets/images/npm.png';
+import Tweet from '../../components/Tweet';
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 304px;
+  grid-gap: 2rem;
+  position: relative;
+  margin-top: 15rem;
+`;
+
+const Section = styled.section`
+  display: grid;
+  grid-template-columns: repeat(3, 300px);
+  grid-gap: 2rem;
+  margin-top: 6.5rem;
+  position: relative;
+  z-index: 2;
+`;
+
+const White = styled.span`
+  color: white;
+`;
+
+const Experiment = () => (
+  <>
+    <Grid>
+      <div>
+        <H2>Experiment Easily</H2>
+        <P
+          muted
+          css={`
+            margin-bottom: 2rem;
+          `}
+        >
+          Try things out to see how they work in reality
+        </P>
+        <P muted>
+          <White>Learn new frameworks</White> and test your understanding{' '}
+        </P>
+        <P muted>
+          <White>Evaluate npm modules</White> to see what works for your project{' '}
+        </P>
+        <P muted>
+          <White>Test out components</White> and create functional examples
+        </P>
+      </div>
+      <Tweet
+        style={`
+          right: -112px;
+          left: auto;
+          width: 416px;
+          margin-top: 0;
+          position: absolute;
+          height: 440px;
+          background: #151515
+        `}
+        tweet={{
+          username: 'gethackteam',
+          job: 'Frontend Dev,eloper Hackteam',
```

Comment:
New Homepage !!!! Looks Really Neat 🔥 💯 

I think there is a small comma issue :)
```suggestion
          job: 'Frontend Developer, Hackteam',
```

Reply:
Ohhh, thank you! File was changed but I updated :)